devsetup/scripts/gen-ansibleee-ssh-key.sh

```diff
@@ -32,14 +32,13 @@ fi
 
 pushd ${OUTPUT_DIR}
 
-if oc get secret dataplane-ansible-ssh-private-key-secret -n ${NAMESPACE} 2>&1 1>/dev/null; then
-    echo "Secret dataplane-ansible-ssh-private-key-secret already exists."
-    echo "Delete it first to recreate:"
-    echo "oc delete secret dataplane-ansible-ssh-private-key-secret"
-    exit 0
-fi
-
 if [ ! -f ${SSH_KEY_FILE} ]; then
+    if oc get secret dataplane-ansible-ssh-private-key-secret -n ${NAMESPACE} 2>&1 1>/dev/null; then
```

Comment:
We don't need this at all here or where it was added before. [1] would allow you to update the secret if exists. We should just remove all this check and instructions.

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/gen-ansibleee-ssh-key.sh#L59-L60

Reply:
I'll close this PR